easybuild/easyconfigs/p/PANDAseq/PANDAseq-2.11-intel-2018a.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'ConfigureMake'
+
+name = 'PANDAseq'
+version = '2.11'
+
+homepage = 'https://github.com/neufeld/pandaseq'
+description = """PANDASEQ is a program to align Illumina reads, optionally with PCR primers embedded in the sequence, and reconstruct an overlapping sequence."""
```

Comment:
@ajaust line should be shorter than 120 characters, so please wrap this, for example:

```python
description = """PANDASEQ is a program to align Illumina reads, optionally with PCR primers embedded
 in the sequence, and reconstruct an overlapping sequence."""